### PR TITLE
4DO_libretro: a 3DO emulator using the libretro API

### PR DESCRIPTION
**Pull request by kwyxz**
This is the libretro port of 4DO, a 3DO emulator based off the FreeDO source code

### CODE REVIEW

File: games-emulation/4do_libretro/4do_libretro-1.3.2.4.20180219git.recipe [hunk 1]
```diff
@@ -0,0 +1,53 @@
+SUMMARY="A port of 4DO, a 3DO emulator to the libretro architecture"
+DESCRIPTION="4DO is an open-source, low-level emulator for the Panasonic 3DO, based on the FreeDO source code."
+HOMEPAGE="http://www.fourdo.com/"
+COPYRIGHT="2011-2018 the 4DO/FreeDO team, the libretro team"
+LICENSE="
+	FreeDO
+	GNU LGPL v3
+	"
```
Comment (fbrosson) [inline games-emulation/4do_libretro/4do_libretro-1.3.2.4.20180219git.recipe]:
```diff
-LICENSE="
-	FreeDO
+LICENSE="FreeDO
-	GNU LGPL v3
-	"
+	GNU LGPL v3"
```
(I'm not 100% sure. I'll do some metrics to see if we really have more recipes with the compact style I'm suggesting.)

File: games-emulation/4do_libretro/4do_libretro-1.3.2.4.20180219git.recipe [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion%~.*}/" \
```
Comment (fbrosson) [inline games-emulation/4do_libretro/4do_libretro-1.3.2.4.20180219git.recipe]: Maybe this is not a good idea, but if you don't want to see the `git` suffix in the core info then:
```diff
-	sed -e "s/@DISPLAY_VERSION@/v${portVersion%~.*}/" \
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion%git}/" \
```

Comment (kwyxz) [reply games-emulation/4do_libretro/4do_libretro-1.3.2.4.20180219git.recipe]: I like it actually.

**Comment by kwyxz** [comment]
I adopted the compact style you proposed for the licenses @fbrosson it seems to work well.

**Pull request merged**